.gitignore

```diff
@@ -130,3 +130,6 @@ dmypy.json
 
 .vscode
 *.DS_Store
+
+
+db-checkbook/historical_cpdb_data/
```

Comment:
I think if you upload the input data to the .library file it will be ignored - so maybe we should move the historical cpdb data there? (Although this totally works too)